slides/lesson1/solution/main_hard_code.tf

```diff
@@ -58,17 +58,23 @@ resource "aws_instance" "web" {
     command = "bash -c 'MAX=10; C=0; until curl -s -o /dev/null ${aws_instance.web.public_dns}; do [ $C -eq $MAX ] && { exit 1; } || sleep 10; ((C++)); done;' || false"
   }
 
-  user_data = "${data.template_file.user_data.rendered}"
+  user_data = data.template_file.user_data.rendered
 
-  vpc_security_group_ids = ["${aws_security_group.web.id}"]
+  vpc_security_group_ids = [aws_security_group.web.id]
 
-  tags = "${merge(var.tags, map("Name", format("%s-%s",data.aws_caller_identity.current.user_id,"ec2")))}"
+  tags = merge(
+    var.tags,
+    {
+      "Name" = format("%s-%s", data.aws_caller_identity.current.user_id, "ec2")
+    },
+  )
 }
 
 resource "aws_route53_record" "dns_web" {
-  zone_id = "${data.aws_route53_zone.current.zone_id}"
+  zone_id = data.aws_route53_zone.current.zone_id
   name    = "${data.aws_caller_identity.current.user_id}.academy.wizeline.dev"
   type    = "A"
   ttl     = 300
-  records = ["${aws_instance.web.public_ip}"]
-}
\ No newline at end of file
+  records = [aws_instance.web.public_ip]
+}
+
```

Comment:
styling: let's remove the extra line

Reply:
we can do this in the next cycle this is not a blocker, thanks for pointing that out

Reply:
This will be done in another PR. Created an issue to keep track of this #33